cinema/management/wait_for_db.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections, DEFAULT_DB_ALIAS
+
+
+class Command(BaseCommand):
+    """
+    Command that waits for the database to become available.
+    """
+
+    def handle(self, *args, **options):
+        """
+        Repeatedly checks the database connection, waits if unavailable.
+        """
+        self.stdout.write("Waiting for database...")
+        max_attempts = 5
+        for attempt in range(max_attempts):
+            connection = connections[DEFAULT_DB_ALIAS]
+            if connection.ensure_connection():
```

Comment:
The 'ensure_connection()' method returns None if the connection to the database was successful, and raises an error if it was unsuccessful. So, checking the result of this method with an 'if' statement is not appropriate. You need to handle the possible exception instead.